requirements.in

```diff
@@ -49,6 +49,7 @@ qrcode
 redis[hiredis]
 reportlab
 requests
+sentry-sdk[flask,celery,sqlalchemy,pure_eval]
```

Comment:
I added it as a regular dependency, since it's more convenient that way: Passing data to the `sentry_sdk` is safe even if it's not configured/initialized, and in that case it never sends anything to a sentry instance, but it avoids having checks whether sentry is installed everywhere...